data/publiccloud/azure_rbac.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash -eu
+set -o pipefail
+#create a role and configure to Resource group and assign role to subscription
+#list, show and delete the created resource
+#Report list of cli's used
+#Role Base Access Control AZURE CLI testing script
+#
+
+#Required AZ setting variable block
+grpname="${1:-oqaclitest}"
+location="${2:-westus}"
+vmname="${3:-oqaclivmss}"
+ssh_key="${4:-oqaclitest-sshkey}"#
+vmximagename="${5:-UbuntuLTS}"
+
+# Variable block
+account="Azure RD"
+spname="oqaclitestsp"
+config_file="azure_rbac.json"
+role="oqaclitest-compute-access"
+asptype="ServicePrincipal"
+cli_cnt=0
+total_cnt=20
+fail_cnt=0
```

Comment:
Yes, if there is any failure we want to come out and it deletes the resource group..
this test count is just to get the idea of how many cli's are covered






[8:56](https://suse.slack.com/archives/D0318C6C1FF/p1678460187493319)
it will be either fully pass or all fail